.openpublishing.redirection.json

```diff
@@ -4193,6 +4193,10 @@
     {
       "source_path": "docs/visual-basic/reference/vb6-support.md",
       "redirect_url": "https://docs.microsoft.com/previous-versions/visualstudio/visual-basic-6/visual-basic-6-support-policy"
+    },
+    {
+      "source_path": "docs/spark/tutorials/ml-sentment-analysis.md",
```

Comment:
We try to keep entries in this file sorted alphabetically by `source_path`

Reply:
Background: We started keeping this sorted because it has worked well to avoid any merge conflicts in this file.

Reply:
Thanks - this makes sense. I didn't realize they were sorted alphabetically.